packages/teleport/src/config.ts

```diff
@@ -18,6 +18,10 @@ import { generatePath } from 'react-router';
 import { merge } from 'lodash';
 
 const cfg = {
+  isEnterprise: false,
+
+  docsVersion: '4.2.8',
```

Comment:
This will be problematic to support as it's easy to forget to update this section every time the doc version change. Instead we need to pass current teleport version and then redirect to the proper docs ourselves. So for this PR, we need to do this:

```jsx
const getDocURLs = (version) => {
   const verprefix = cfg.isEnterprise ? 'e' : 'oss';
   const withUTM = url => $(url)?utm_source=teleport&utm_medium=${verprefix}${version}`;
 
   return {
     troubleshooting:  withUTM('https://gravitational.com/teleport/docs/admin-guide/#troubleshooting'),
     faq:  withUTM('https://gravitational.com/teleport/docs/admin-guide/#faq'),
   }
}
```
We can place it in the same file as component - easy to move, easy to delete!

Reply:
revised